app/views/admin/products_v3/_table.html.haml

```diff
@@ -11,16 +11,16 @@
   %col{ width:"8%", style: "max-width:8em" }
   %thead
     %tr
-      %th.align-left= t('admin.product.name')
-      %th.align-right= t('admin.sku')
-      %th.align-right= t('admin.unit')
-      %th.align-right= t('admin.price')
-      %th.align-right= t('admin.on_hand')
-      %th.align-left= t('admin.producer')
-      %th.align-left= t('admin.category')
-      %th.align-left= t('admin.tax_category')
-      %th.align-left= t('admin.inherits_properties')
-      %th.align-right= t('admin.available_on')
+      %th.align-left= t('admin.products_page.columns.name')
+      %th.align-right= t('admin.products_page.columns.sku')
+      %th.align-right= t('admin.products_page.columns.unit')
+      %th.align-right= t('admin.products_page.columns.price')
+      %th.align-right= t('admin.products_page.columns.on_hand')
+      %th.align-left= t('admin.products_page.columns.producer')
+      %th.align-left= t('admin.products_page.columns.category')
+      %th.align-left= t('admin.products_page.columns.tax_category')
+      %th.align-left= t('admin.products_page.columns.inherits_properties')
+      %th.align-right= t('admin.products_page.columns.available_on')
```

Comment:
`available_on` column has been deleted in the meantime : https://github.com/openfoodfoundation/openfoodnetwork/pull/11136

Reply:
Ok, deleted ;)